src/shared/utils/fetchApi.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+
+async function fetchApi({ url, method, body, propertyName }) {
+  const defaultToken =
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiIxN2I2MDZjMS05OTk3LTRlNWYtODBmMC0xYWEyN2VlOGE1NTMiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20iLCJpYXQiOjE2NDc2MjgwODR9.CcWEd_xGSx_T1lCyeyyys4JLCVuWR5ThmVVtgi0qWmE";
+   const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiIyYWNmMTkyZi0zYzk3LTQ1NWMtYjEzNi0yZmJkMzY0ODliYjEiLCJlbWFpbCI6InByaXlhQGdtYWlsLmNvbSIsImlhdCI6MTY0NzY5MzQxNX0.9vgp3G1Vrq-pHegSc4hfwA-i4EsKiKdpqPJT-ERY0IQ"
```

Comment:
token will change on every user session so providing a defaultToken won't solve your problem. Try providing the token from localstorage only.

Reply:
Yeah i will do  this when I will do the authentication .